.github/workflows/ngsa.yaml

```diff
@@ -104,33 +104,31 @@ jobs:
       run: |
         dotnet tool install -g webvalidate
 
-    - name: Start helium container
+    - name: Start NGSA container
       if: ${{ env.COSMOS_SET == 'true' }}
       run: |
         # run the app
         docker run -d --name ngsa -p 4120:4120 test
         
         echo "Waiting for web server to start ..."
         sleep 10
```

Comment:
could we move this down a line and do a sleep $wait_time?  Then we would only have to update once if we wanted to change.